rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,421 @@
+
+### Timebox for Roadmaps
+
+All timeboxes are noted as business days. Business days are included regardless of national holidays.
+
+Overlapping timeboxes denote concurrent work.
+
+Time boxes with a range of days indicate that the work MAY be completed at any point during the timebox, but MUST be completed no later than the final day of the timebox.
+
+**Note**: *Input from the Technical Architecture team is integrated on a quarterly basis during Release planning.*
+
+Day 1 in the Roadmap timeline begins in the third quarter.
+
+* **Day 1 - Day 25: [Gathering and reviewing input](#gathering-and-reviewing-input)**
+  * Prerequisite: Science Roadmap
+  * Day 1-25: DCP User Survey 
+  * Day 1-10: Roadmap Retrospective
+* **Day 1 - Day 40: [Drafting the roadmap](#drafting-the-roadmap)**
+  * Day 1-10: Identify a Roadmap Shepherd
+  * Day 25-30: Create themes
+  * Day 31: Collate themes
+  * Day 32-40: Refine themes
+* **Day 40 - Day 55: Receiving feedback**
+  * Day 40: Publish draft Roadmap
+  * Day 41-50: Community Review period 
+  * Day 51-55: Complete final edits
+* **Day 55 - Day 65 : [Iterating with Oversight](#iterating-with-oversight)**
+  * Day 55: Send Roadmap to Oversight
+  * Day 56-65: Review and edit roadmap with Oversight
+  * Day 65: Receive Roadmap approval from Oversight
+  * Day 65: Communicate approved Roadmap 
+
+### Gathering and Reviewing Input
+[Gathering and Reviewing Input]:#gathering-and-reviewing-input
+
+#### Science Roadmap
+
+Oversight MUST provide an annual Science Roadmap to Project Leads to be reviewed at the next Oversight meeting following its receipt.
+
+Project Leads MUST clarify outstanding questions about the Science Roadmap with Oversight within two weeks.
+
+#### Technical Architecture Objectives 
+
+Through a process of their own choosing, the Technical Architecture team MAY document critical architectural issues and technical debt in a Technical Objectives Informational RFC for consideration during each Release. The RFC SHOULD outline technical strategies and estimate cost (such as two engineers at 20%) to realize its objectives. 
+
+The RFC MUST be approved prior to [Clarifying the Next Release](#clarifying-the-next-release) for consideration.
+
+#### DCP User Survey
+
+User Experience (UX) MUST create a DCP user survey on an annual basis to collect input for the Product Roadmap. The audience for the survey are the Users of the DCP.
+
+UX MUST review the proposed survey with the DCP community and incorporate suggestions as appropriate.
+
+The survey MUST be announced to the membership of the Human Cell Atlas (through its the Registration mailing list) with a minimum of a two week last call for responses.
+
+At the conclusion, UX MUST summarize insights from the survey and present at the next DCP PM meeting. Following a one week review to incorporate feedback, the survey MUST be shared by UX with the membership of the Human Cell Atlas and announced on the *HumanCellAtlas #dcp* slack channel.
+
```

Comment:
A few thoughts on the annual user survey section: 

1. I feel this survey may not be necessary for 2019, depending on timing of this roadmapping, since we may not have any "Users of the DCP" beyond the community members we have already collected feedback from in the Beta (and the broader community we've spoken to in other research initiatives).  
2. To make sure this survey work is actionable and useful, I think we need more alignment here about _how_ the results of the survey are expected to impact the Product Roadmap.  What types of issues or insights are we hoping to shed light on with this survey? 
3. I think the PM team SHOULD also incorporate feedback that the UX team has gathered from other avenues (interviews, other UX projects - not just this one survey) into the roadmapping process. 
4. I am having trouble understanding the overall timeline prescribed for this survey.  I want to make sure it is realistic - we will likely need 2 weeks to develop and test the survey and get all stakeholders' buy-in, plus the 2 weeks to collect responses, then 1 week to create an initial analysis.  Above, you specify that we then need another week to incorporate PM feedback before sharing the survey - so that is 6 weeks total.

Reply:
I'll defer to @CaraLMason for most of your comments. The authors are in agreement about deferring the user survey to 2020 for the reasons that you've outlined. We could add a ***Note:** to that point in the RFC.

Reply:
@ambrosejcarr @adriennes Ah, thanks for clarifying. We can start day 1 at whatever point is needed for the UX survey. My plan in including timeboxes was to indicate the length of time that we are giving to steps in the process, in order to manage dependencies and avoid slippage.

Reply:
RE @adriennes (3) - _PM team SHOULD also incorporate feedback that the UX team has gathered from other avenues_ - I recommend adding a closing sentence to the DCP User Survey section - _UX MAY also share critical UX insights from other sources independent of the Survey for consideration during the creation of the Roadmap._ unless UX would prefer to commit to producing their Insights as Informational RFC(s) similar to the Technical Objectives. @adriennes and @gabsie ?

Reply:
@brianraymor  - I like this edit - I think a single report of insights is the right format here, not informational RFCs.  
@CaraLMason and @ambrosejcarr  - yes, the survey by my rough calculation will need at least 1 more week than scheduled here.  I agree that timeboxes are helpful to prevent slippage, so I want us to start off with a realistic timebox so we don't immediately miss our deadline.  I think it could be ok to say that UX preparation for the survey needs to begin before this process, though, and by the start of the roadmapping process, UX needs to have a draft survey prepared. 
@CaraLMason  - On reflection, I'm ok with leaving the description of the survey here open ended as it is, especially as we add in the comments about combining with other UX insights.  That will give us flexibility to implement whatever is needed each year.

Reply:
I have an updated Roadmap timeline (30 days for UX survey) in review with @CaraLMason.

Reply:
@adriennes @brianraymor Reviewed the proposed timeline from Brian and it looks right to me.

Reply:
Timeline updated in aa50a0a653e71958e8aa4b90d587979e8b9a17a8.

Reply:
Resolving since I believe that all the points are now addressed.